pinot-spi/src/test/java/org/apache/pinot/spi/filesystem/LocalPinotFSTest.java

```diff
@@ -188,7 +188,7 @@ public void testFS()
     localPinotFS.touch(nonExistingFile.toURI());
     Assert.assertTrue(nonExistingFile.exists());
     long currentTime = System.currentTimeMillis();
-    Assert.assertTrue(localPinotFS.lastModified(nonExistingFile.toURI()) < currentTime);
+    Assert.assertTrue(localPinotFS.lastModified(nonExistingFile.toURI()) <= currentTime);
```

Comment:
Perhaps add sleep to ensure `<` can be used?

Reply:
I don't see in what `<=` is not a valid assertion. I'm not a fan of adding sleeps in a codebase :/

Reply:
I dont believe mayank was asking to add sleep in code base. I think he was suggesting a sleep before line 190. Say, `Thread.sleep(10)`

Reply:
That's what I understood too. One or the other I don't really mind. I just think that an equals value is still a valid assertion in that context and would save us an irrelevant `Thread.sleep`. Just my 2 cents :)